test/filters/oidc/jwks_storage_test.cc

```diff
@@ -0,0 +1,69 @@
+#include "src/filters/oidc/jwks_storage.h"
+
+#include "gtest/gtest.h"
+
+namespace authservice {
+namespace filters {
+namespace oidc {
+
+namespace {
+// A good public key based on above private key
```

Comment:
```suggest
// A valid public JWK key for JWT verification.
const char valid_jwks_public_key_ [] = R"(
```

Reply:
signing key means private key in these contexts. I think we are testing public key part.